cider/exec/nextgen/jitlib/llvmjit/LLVMJITModule.cpp

```diff
@@ -126,7 +126,9 @@ JITFunctionPointer LLVMJITModule::createJITFunction(
 
 void LLVMJITModule::finish() {
   // TODO (bigPYJ1151): Refactor Debug information.
```

Comment:
Refactor to make it configurable to enrich content? @bigPYJ1151  Do we still this TODO since we make it configurable via ```co```?

Reply:
Yes, module's IR, query_func's IR and ASM should be placed properly and configurable.